src/SFA.DAS.Campaign.Web/Controllers/ApprenticeController.cs

```diff
@@ -16,8 +27,15 @@ public IActionResult WhatAreTheBenefitsToMe()
             return View();
         }
         [Route("find-an-apprenticeship")]
-        public IActionResult FindAnApprenticeship()
+        public async Task<IActionResult> FindAnApprenticeship()
         {
+            var health = await _vacancyServiceApiHealthCheck.CheckHealthAsync(new HealthCheckContext(), CancellationToken.None);
```

Comment:
So instead of handling the rare event of the call failing, we're calling out the the APIs many times _every single time_ the search page is requested?